scripts/volumetric_evaluation/evaluate_lucchi.py

```diff
@@ -0,0 +1,135 @@
+import os
+import h5py
+import itertools
+import pandas as pd
+import numpy as np
+from math import floor
+from typing import Union
+
+from skimage.measure import label
+
+from micro_sam import util
+from micro_sam.inference import batched_inference
+from micro_sam.prompt_generators import PointAndBoxPromptGenerator
+from micro_sam.multi_dimensional_segmentation import segment_mask_in_volume
+
+from elf.evaluation import mean_segmentation_accuracy
+
+
+def get_raw_and_label_volumes(volume_path):
+    with h5py.File(volume_path, "r") as f:
+        raw = f["raw"][:]
+        label = f["labels"][:]
+
+    return raw, label
+
+
+def segment_lucchi_from_slices(
+    input_path: str,
+    model_type: str,
+    checkpoint: str,
+    embedding_path: str,
+    iou_threshold: float = 0.8,
+    projection: Union[str, dict] = "mask",
+    box_extension: Union[float, int] = 0.025
+):
+    _get_model = util.get_sam_model if checkpoint is None else util.get_custom_sam_model
+    predictor = _get_model(model_type=model_type, checkpoint_path=checkpoint)
+
+    test_volume_path = os.path.join(input_path, "lucchi_test.h5")
+    volume, labels = get_raw_and_label_volumes(test_volume_path)
+
+    # precompute embeddings
+    image_embeddings = util.precompute_image_embeddings(predictor, volume, save_path=embedding_path, ndim=3)
+
+    # let's get the instances from the label volume
+    instance_labels = label(labels)
```

Comment:
We should assume that we get instance labels passed to the function.
(So for lucchi you can do this beforehand)